daemon/config/config.go

```diff
@@ -286,9 +287,9 @@ func New() *Config {
 			MaxConcurrentDownloads: DefaultMaxConcurrentDownloads,
 			MaxConcurrentUploads:   DefaultMaxConcurrentUploads,
 			MaxDownloadAttempts:    DefaultDownloadAttempts,
-			Mtu:                    DefaultNetworkMtu,
 			NetworkConfig: NetworkConfig{
 				NetworkControlPlaneMTU: DefaultNetworkMtu,
+				Mtu:                    DefaultNetworkMtu,
```

Comment:
Moving this looks to be breaking this test
https://github.com/moby/moby/blob/f34567bf41ad65102a8b1f05496dc92b500a3056/daemon/config/config_test.go#L255-L258
and this test
https://github.com/moby/moby/blob/f34567bf41ad65102a8b1f05496dc92b500a3056/daemon/config/config_test.go#L411-L414

so probably it should be better to add new flag like `--default-mtu` like there looks to be couple of existing parameters like those already and add test cases for that (at least MTU with and without that value) as maintainer always like to see those.

Reply:
I went for `--network-default-mtu` - WDYT?

Reply:
Fine for me.